src/common/contexts/TokenInformationContext/TokenInformationContext.tsx

```diff
@@ -143,10 +143,10 @@ export const TokenInformationContextProvider = ({ children }: { children: React.
         tokenId,
         tokenRegistryAddress,
         initialize,
-        holder,
-        beneficiary,
-        approvedBeneficiary,
-        approvedHolder,
+        holder: holder?.[0],
+        beneficiary: beneficiary?.[0],
+        approvedBeneficiary: approvedBeneficiary?.[0],
+        approvedHolder: approvedHolder?.[0],
```

Comment:
Not sure why but typescript says its this.

Reply:
I remind having this when I upgraded oa-verify to ethers v5 as weill. However when using token-registry or document-store package, no problem anymore :thinking: